### PR TITLE
Add missing ids

### PR DESCRIPTION
**Pull request by nazli-stripe**
Adding a couple missing IDs to `ids.rs`

### CODE REVIEW

**Comment by arlyon** [comment]
This is great, thank you @nazli-stripe! :) Going to wait for CI for the sake of process and merge this. :tada:

**Review by arlyon — CHANGES_REQUESTED** [review]

File: src/ids.rs [hunk 1]
```diff
@@ -512,9 +512,9 @@ def_id!(IssuingDisputeId, "idp_");
 def_id!(IssuingTransactionId, "ipi_");
 def_id!(OrderId, "or_");
 def_id!(OrderReturnId, "orret_");
-def_id!(MandateId: String); // TODO: Figure out what prefix this id has
+def_id!(MandateId: "mandate_");
```
[the remark this answers is not in the document]
Comment (nazli-stripe) [reply src/ids.rs]: Accepted suggestion, thanks for catching!

File: src/ids.rs [hunk 2]
```diff
@@ -484,7 +484,7 @@ def_id!(CardTokenId, "tok_");
 def_id!(ChargeId, "ch_" | "py_"); // TODO: Understand (and then document) why "py_" is a valid charge id
 def_id!(CheckoutSessionId, "cs_");
 def_id!(CheckoutSessionItemId: String); // TODO: Figure out what prefix this id has
-def_id!(ConnectCollectionTransferId: String);
+def_id!(ConnectCollectionTransferId: "connct_");
```
Comment (arlyon) [inline src/ids.rs:487]:
```suggestion
def_id!(ConnectCollectionTransferId, "connct_");
```

File: src/ids.rs [hunk 3]
```diff
@@ -512,9 +512,9 @@ def_id!(IssuingDisputeId, "idp_");
 def_id!(IssuingTransactionId, "ipi_");
 def_id!(OrderId, "or_");
 def_id!(OrderReturnId, "orret_");
-def_id!(MandateId: String); // TODO: Figure out what prefix this id has
+def_id!(MandateId: "mandate_");
```
Comment (arlyon) [inline src/ids.rs:515]:
```suggestion
def_id!(MandateId, "mandate_");
```

File: src/ids.rs [hunk 4]
```diff
@@ -512,9 +512,9 @@ def_id!(IssuingDisputeId, "idp_");
...
 def_id!(PaymentIntentId, "pi_");
-def_id!(PaymentLinkId: String);
+def_id!(PaymentLinkId: "plink_");
```
Comment (arlyon) [inline src/ids.rs:517]:
```suggestion
def_id!(PaymentLinkId, "plink_");
```

File: src/ids.rs [hunk 5]
```diff
@@ -537,7 +537,7 @@ def_id!(
 def_id!(PersonId, "person_");
 def_id!(PlanId: String); // N.B. A plan id can be user-provided so can be any arbitrary string
 def_id!(PlatformTaxFeeId: String);
-def_id!(PriceId: String); // TODO: Figure out what prefix this id has
+def_id!(PriceId: "price_");
```
Comment (arlyon) [inline src/ids.rs:540]:
```suggestion
def_id!(PriceId, "price_");
```

**Comment by arlyon** [comment]
Rebased this and merged it here: https://github.com/arlyon/async-stripe/pull/243 :) Thanks for the contribution

**Pull request closed without merging**